### ISSUE TITLE
Machine Polish (pre GCC-2017)

### ISSUE THREAD

**Issue by jmchilton** [galaxyproject/planemo-machine#85]
## Done

- [x] Nginx should not say "forbidden" while starting up - it should say "Galaxy is unavailable - it may be starting..." (https://github.com/galaxyproject/ansible-galaxy-extras/blob/master/templates/nginx.conf.j2)
- [x] enable IEs in default config (https://github.com/galaxyproject/planemo-machine/blob/master/group_vars/all#L115)
- [x] change prompt (maybe just cwd, rather than default whole path ubuntu uses) (https://github.com/galaxyproject/planemo-machine/blob/master/roles/galaxyprojectdotorg.devbox/tasks/main.yml)
- [x] Double power indicators - remove indicator-power package duplicated by xubuntu.
- [x] Double date indicators - remove indicator-datetime package duplicated by xubuntu.
- [x] Do you have to restart galaxy to see the tool? Hot load? (This is a bug - those tools should definitely show up.)
- [x] Munge does not work in 16.04 image. Works in Docker container - but not in VM. Maybe conflicts between ways of starting this up?
- [x] Configure proxy - it was configured. Fixed in #95 - now tested to prevent regressions.
- [x] Node is cannot install proxy on 16.04. Fixed in #95 - now tested to prevent regressions.
- [x] npm install proxy stuff (This is probably done by ansible-galaxy-extras now? https://github.com/galaxyproject/ansible-galaxy-extras/blob/master/defaults/main.yml#L158 - might have to witch defaults to enable it)
- [x] ~~missing  creation of a tool_dependencies.xml file which would be required for the seqkt and bwa tools to actually work if downloaded from the toolshed (place in planemo maybe?)~~ (Don't need this anymore.)
- [x] disable screen saver
- [x] link to the galaxy log on the desktop of the virtual machine. (https://github.com/galaxyproject/planemo-machine/blob/master/roles/galaxyprojectdotorg.devbox/tasks/main.yml)
- [x] Turn off Terminal scrollback limits. (https://github.com/galaxyproject/planemo-machine/blob/master/roles/galaxyprojectdotorg.devbox/tasks/main.yml)
- [x] Turn off Terminal scroll on update. (https://github.com/galaxyproject/planemo-machine/blob/master/roles/galaxyprojectdotorg.devbox/tasks/main.yml)
- [x] change font size for terminal (https://github.com/galaxyproject/planemo-machine/blob/master/roles/galaxyprojectdotorg.devbox/tasks/main.yml)

**Comment by hexylena** [comment]
![utvalg_883](https://cloud.githubusercontent.com/assets/458683/16363723/f25c8a4a-3bc2-11e6-814a-d588ebecf672.png)

two battery icons are twice as useful ;)

**Comment by hexylena** [comment]
@bgruening https://github.com/jmchilton/galaxy-downloads the databases are so small, can we include that so planemo inside planemo machine doesn't have to hit the network at all? Also maybe pre-run a planemo profile?

**Comment by bgruening** [comment]
I would like to see this database as part of planemo :)

**Comment by martenson** [comment]
I checked off the things done in #100

**Comment by jmchilton** [comment]
Moved 6 more things to the Done list based on previous fixes and fixes in #105. Including progress on IE, duplicate indicators, shortening command prompt, tool reloading, and munge being broken. 

Additionally I moved two more things to a new issue #104 - related to IEs and API since I don't think we will have an API training session at this years GCC and I don't think any training sessions will require us to pre-fetch IEs.

Only one thing remains and that is the poor error handling in nginx - I'm going to start trying to work on the error docs after #106 clears testing (the ansible-galaxy role has added such error docs lately that we can leverage from here).
